stories/menu.stories.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Same here, we need to add a file header.

Reply:
Updated @eugene-manuilov

Reply:
Thanks!